src/reactjs/useBaseQuery.ts

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import { useSyncExternalStore } from 'use-sync-external-store/shim/index.js'
+import { useSyncExternalStore } from './useSyncExternalStore'
```

Comment:
we have other imports from the shim as well that need to be adapted please

Reply:
done